main.go

```diff
@@ -218,6 +219,15 @@ func fetchPages(team, since, until string) ([]*page, error) {
 	var pages []*page
 
 	for _, p := range incResp.Incidents {
+		if tagFilters != nil && len(tagFilters) > 0 {
+			if matched, err := pagerdutyIncidentMatchesTags(client, p.Id, tagFilters); err != nil {
+				fmt.Fprintf(os.Stderr, "Could not fetch tags for incident %s, skipping: %v\n", p.Id, err)
+				continue
+			} else if !matched {
+				continue
+			}
+		}
```

Comment:
This might be easier to read if the `tagFilters != nil && len(tagFilters) > 0` is inside `pagerdutyIncidentMatchesTags`, thoughts? Also splitting the if/else into two ifs for readability?

Reply:
Done!